### PR TITLE
Chore/og

### PR DESCRIPTION
**Pull request by abdullah50053**

### CODE REVIEW

**Comment by nam-t24** [comment]
I'll work on this later cuz I can't get it to work rn either

**Comment by DedsecKnight** [comment]
What is this branch about?

**Comment by nam-t24** [comment]
Adding the HackUTD logo as the thumbnail image for social media previews. It's under pages/_app.tsx

File: pages/index.tsx [hunk 1]
```diff
@@ -529,7 +539,7 @@ export default function Home(props: {
               </a>
               {/* !change */}
               <a
-                href="https://github.com/acmutd/hackportal"
+                href="https://github.com/acmutd/hackutd-ix-site"
```
Comment (NickBurnett) [inline pages/index.tsx:542]: Are we sure about hardcoding this?

File: pages/index.tsx [hunk 2]
```diff
@@ -596,15 +606,15 @@ export default function Home(props: {
               Contact Us
             </a>
             <a
-              href="https://github.com/acmutd/hackportal"
+              href="https://github.com/acmutd/hackutd-ix-site"
```
Comment (NickBurnett) [inline pages/index.tsx:609]: Same thing about hardcoding here. Maybe it's worth moving to a "constants" file or just making it available in the environment.

**Review by NickBurnett — APPROVED** [review]
Nonblocking comments.

**Pull request merged**